src/internal.c

```diff
@@ -5216,6 +5216,8 @@ static int Receive(WOLFSSL* ssl, byte* buf, word32 sz)
                             return WANT_READ;
                         }
                     }
+				#elif defined(FREERTOS_TCP)
+					return WANT_READ;	/* want read, call again    */
```

Comment:
Chris,
I will turn RECV_FUNCTION/SEND_FUNCTION into simple 1-to-1 macro's
TranslateReturnCode() will translated the return codes and set errno.
I'm not sure if FreeRTOS users want to have a errno.h in their projects.
The errno variable is often not thread-aware,
FreeRTOS does have 'variables' that are local to a task. But those can only be accessed by functions. FreeRTOS+FAT does use this alternative for errno.